docker/publish-by-arch.sh

```diff
@@ -0,0 +1,248 @@
+  docker manifest create --insecure --amend inlong/agent:latest \
+    inlong/agent:latest-arm64 inlong/agent:latest-x86
+  docker manifest create --insecure --amend inlong/dataproxy:latest \
+    inlong/dataproxy:latest-arm64 inlong/dataproxy:latest-x86
+  docker manifest create --insecure --amend inlong/tubemq-manager:latest \
+    inlong/tubemq-manager:latest-arm64 inlong/tubemq-manager:latest-x86
+  docker manifest create --insecure --amend inlong/tubemq-all:latest \
+    inlong/tubemq-all:latest-arm64 inlong/tubemq-all:latest-x86
+  docker manifest create --insecure --amend inlong/tubemq-build:latest \
+    inlong/tubemq-build:latest-arm64 inlong/tubemq-build:latest-x86
+  docker manifest create --insecure --amend inlong/dashboard:latest \
+    inlong/dashboard:latest-arm64 inlong/dashboard:latest-x86
+  docker manifest create --insecure --amend inlong/tubemq-cpp:latest \
+    inlong/tubemq-cpp:latest-arm64 inlong/tubemq-cpp:latest-x86
+  docker manifest create --insecure --amend inlong/audit:latest \
+    inlong/audit:latest-arm64 inlong/audit:latest-x86
+
+  docker manifest create --insecure --amend inlong/manager:${MVN_VERSION} \
+    inlong/manager:${MVN_VERSION}-arm64 inlong/manager:${MVN_VERSION}-x86
+  docker manifest create --insecure --amend inlong/agent:${MVN_VERSION} \
+    inlong/agent:${MVN_VERSION}-arm64 inlong/agent:${MVN_VERSION}-x86
+  docker manifest create --insecure --amend inlong/dataproxy:${MVN_VERSION} \
+    inlong/dataproxy:${MVN_VERSION}-arm64 inlong/dataproxy:${MVN_VERSION}-x86
+  docker manifest create --insecure --amend inlong/tubemq-manager:${MVN_VERSION} \
+    inlong/tubemq-manager:${MVN_VERSION}-arm64 inlong/tubemq-manager:${MVN_VERSION}-x86
+  docker manifest create --insecure --amend inlong/tubemq-all:${MVN_VERSION} \
+    inlong/tubemq-all:${MVN_VERSION}-arm64 inlong/tubemq-all:${MVN_VERSION}-x86
+  docker manifest create --insecure --amend inlong/tubemq-build:${MVN_VERSION} \
+    inlong/tubemq-build:${MVN_VERSION}-arm64 inlong/tubemq-build:${MVN_VERSION}-x86
+  docker manifest create --insecure --amend inlong/dashboard:${MVN_VERSION} \
+    inlong/dashboard:${MVN_VERSION}-arm64 inlong/dashboard:${MVN_VERSION}-x86
+  docker manifest create --insecure --amend inlong/tubemq-cpp:${MVN_VERSION} \
+    inlong/tubemq-cpp:${MVN_VERSION}-arm64 inlong/tubemq-cpp:${MVN_VERSION}-x86
+  docker manifest create --insecure --amend inlong/audit:${MVN_VERSION} \
+    inlong/audit:${MVN_VERSION}-arm64 inlong/audit:${MVN_VERSION}-x86
+  echo "End pushing manifest"
+}
+
+help() {
+  cat <<EOF
+Usage: ./publish-by-arch.sh [option] <arch>
+where arch is one of:
+  x86
+  aarch64
+Options:
+  -b        Add build operation before publish. Build docker images by arch.
+  -t        Add tag operation before publish. Add arch after version.
+  -p        Publish images according to docker registry information.
+  -m        Push manifest. This option doesn't need arch.
+  -h        Show help information.
```

Comment:
Would it be clearer?
```suggestion
  -b, --build           Add build operation before publish. Build docker images by arch.
  -t, --tag             Add tag operation before publish. Add arch after version.
  -p, --publish         Publish images according to docker registry information.
  -m, --manifest        Push manifest. This option doesn't need arch.
  -h, --help            Show help information.
```

```shell
  if [ "$arg" = "-b" ] || [ "$arg" = "--build" ]; then
    NEED_BUILD=true
  elif [ "$arg" = "-t" ] || [ "$arg" = "--tag" ]; then
    NEED_TAG=true
  elif [ "$arg" = "-m" ] || [ "$arg" = "--manifest" ]; then
    NEED_MANIFEST=true
  elif [ "$arg" = "-p" ] || [ "$arg" = "--publish" ]; then
    NEED_PUBLISH=true
  elif [ "$arg" = "-h" ] || [ "$arg" = "--help" ]; then
    help
    exit 0
```